packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -1192,6 +1210,53 @@ export class Stack extends Construct implements ITaggable {
     return makeStackName(ids);
   }
 
+  private determineImportValue(exportedValue: any) {
+    const resolvable = Tokenization.reverse(exportedValue);
+    if (!resolvable || !Reference.isReference(resolvable)) {
+      throw new Error('exportValue: either supply \'name\' or make sure to export a resource attribute (like \'bucket.bucketName\')');
+    }
+
+    // "teleport" the value here, in case it comes from a nested stack. This will also
+    // ensure the value is from our own scope.
+    const exportable = getExportable(this, resolvable);
+
+    // Ensure a singleton "Exports" scoping Construct
+    // This mostly exists to trigger LogicalID munging, which would be
+    // disabled if we parented constructs directly under Stack.
+    // Also it nicely prevents likely construct name clashes
+    const exportsScope = getCreateExportsScope(this);
+
+    // Ensure a singleton CfnOutput for this value
+    const resolved = this.resolve(exportable);
+    const id = 'Output' + JSON.stringify(resolved);
+    const exportName = generateExportName(exportsScope, id);
+
+    if (Token.isUnresolved(exportName)) {
+      throw new Error(`unresolved token in generated export name: ${JSON.stringify(this.resolve(exportName))}`);
+    }
+
+    const exportIsAList = isExportAList(exportedValue, resolved);
+    const delimiter = 'CDK-string-list-export-delimiter';
```

Comment:
Ha 😆 slightly shorter will do, methinks.

How about `'||'` ?

Reply:
Originally I made it that long to avoid collisions with customer code, but since the exports have to be references of some kind, it's actually not possible for `||` to be used in the `exportedValue`. I'll make this change.